templates/footer.php

```diff
@@ -0,0 +1,3 @@
+<footer>
+    
+</footer>
```

Comment:
is dit bestand nog nodig?

Reply:
Ja, het is voor wat ruimte onderaan de pagina